ads/vendors/verizonmedia.md

```diff
@@ -0,0 +1,54 @@
+<!---
+Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```

Comment:
```suggestion
Copyright 2021 The AMP HTML Authors. All Rights Reserved.
```

Reply:
updated and updated the verizonmedia.js copyright too
Thanks
Ed